README.md

```diff
@@ -112,3 +112,13 @@ To run the app in a VM:
 4. Run the playbook with `ansible-playbook ansible_playbook.yml -i ansible_inventory.ini`
    - you will be prompted to enter the relevant details from your local .env file
    - after successfully running the playbook, you should be able to run the app by entering `MANAGED-NODE-IP-ADDRESS:5000` in a browser
+
+## Building and pushing to docker hub
```

Comment:
Minor, but worth noting that if you are expecting people to also hit the Azure Webhook endpoint to actually trigger a deployment then it would be good to also document that briefly!